src/Discovery/DiscoveryListView.tsx

```diff
@@ -165,7 +167,7 @@ const DiscoveryListView: React.FunctionComponent<Props> = (props: Props) => {
                     props.setModalVisible(true);
                   }}
                 >
-                  {renderValue(studyPreviewText)}
+                  {studyPreviewTextArray.forEach((item: string | undefined) => renderValue(item))}
```

Comment:
```suggestion
                  {studyPreviewTextArray.map((item: string | undefined) => renderValue(item))}
```
I think this is the way to get them rendered